test/unit/local/steps/dispatch.js

```diff
@@ -307,6 +307,63 @@ describe('core/local/steps/dispatch.loop()', function () {
         })
       })
     })
+  })
+
+  context('when buffer contains a overwriting renamed file event', () => {
+    const filePath = 'foo'
+    const newFilePath = 'bar'
+    const md5sum = crypto.createHash('md5').update('').digest().toString('base64')
+
+    beforeEach(() => {
+      buffer.push([
+        builders
+          .event()
+          .action('renamed')
+          .kind('file')
+          .oldPath(filePath)
+          .path(newFilePath)
+          .ino(1)
+          .md5sum(md5sum)
```

Comment:
`.data('')`? (so we can skip the `const md5sum = ...` line)

Reply:
I added the `data()` helper although I'm not a big fan since events themselves don't have data.